opm/autodiff/FlowMain.hpp

```diff
@@ -724,7 +724,7 @@ namespace Opm
                     fullReport.reportParam(tot_os);
                 }
             } else {
-                output_writer_->writeInit( simtimer, geoprops_->nnc() );
+                output_writer_->writeInit( geoprops_->nnc() );
```

Comment:
... except now it should use the nonCartesianNeighbors() method instead of nnc().